en/advanced_features/satcom_roadblock.md

```diff
@@ -65,29 +65,25 @@ The module is now ready to be used with PX4.
 
 ### Software
 
-Flash the Pixhawk with the latest PX4 master. 
-To activate either set the [SYS_COMPANION](../advanced_config/parameter_reference.md#SYS_COMPANION) parameter to `Iridium Telemetry (115200 baud, 8N1)` or add the following lines to the **extras.txt** on the SD card:
-
-```sh
-# add a sleep here to make sure that the module is powered
-usleep 200000
-
-#driver
-if iridiumsbd start -d SERIAL_PORT
-then
-	# start the iridium mavlink instance
-	mavlink start -d /dev/iridium -m iridium -b 115200
-
-	# Sleep a bit to give Mavlink time to set up
-	usleep 100000
-
-else
-	echo "WARN: Iridiumsbd driver not started, reboot"
-	tune_control play -m "ML<<CP4CP4CP4CP4CP4"
-fi
+[Configure the serial port](../peripherals/serial_configuration.md) on which the RockBlock module will run using [ISBD_CONFIG](../advanced_config/parameter_reference.md#ISBD_CONFIG).
+<!-- There is no need to set the baud rate for the port, as this is configured by the driver. -->
+The baud rate for the connected port should be set to: 115200.
+
+> **Note** If `ISBD_CONFIG` is not available in *QGroundControl* then you will need to [add the driver to the firmware](#firmware).
+
+### Firmware Setup {#firmware}
+
+Flash the Pixhawk with the latest PX4 master.
```

Comment:
@bkueng When will this go into a release (specifically the serial port changes)?

Reply:
Next release but I guess that does not help you. I don't know more.

Reply:
Thanks. We'll just update this next time I arrive if there has been a release.